File: Frontend/src/app/catalogo/producto/producto.ts
import {CantidadProducto} from "../cantidad-producto";
import {ProductoCatalogo} from "../producto-catalogo/producto-catalogo";

export class Producto {
  id: number;
  precioPorUnidad: number;
  cantidadDisponible: number;
  cantidadProducto: any;
  productoCatalogo: any;
}

File: Backend/MarketPlace_Backend/marketplace/models/producto.py
from django.db import models
from cantidad_producto import CantidadProducto 
from producto_catalogo import producto_catalogo as r_producto_catalogo

class producto(models.Model):
    precio_por_unidad = models.FloatField(blank=False, null = False)
    cantidad_disponible = models.IntegerField(blank=False, null = False)

    cantidad_producto = models.ManyToManyField(CantidadProducto)
    producto_catalogo = models.ForeignKey(r_producto_catalogo, on_delete=models.CASCADE)

